manage_breast_screening/templates/wizard_step.jinja

```diff
@@ -0,0 +1,60 @@
+{% extends 'layout-app.html' %}
+{% from 'components/radios/macro.jinja' import radios %}
+{% from 'components/button/macro.jinja' import button %}
+{% from 'components/fieldset/macro.jinja' import fieldset %}
+{% from 'components/back-link/macro.jinja' import backLink %}
+
+{% block beforeContent %}
+{#- Using javascript temporarily - this should be replaced with proper URLs -#}
+{{ backLink({
+  "href": "javascript:history.back()",
+  "text": "Go back"
+}) }}
+{% endblock beforeContent %}
+
+{% block content %}
+<h1>{{step.title}}</h1>
+
+{% if form %}
+<form action="{{request.path}}" method="POST">
+{% block form %}
+{% endblock form %}
+
+
+{{ radios({
```

Comment:
I think we need a better way to integrate the macros with django, this is using [manual form rendering](https://docs.djangoproject.com/en/5.2/topics/forms/#rendering-fields-manually) but it's going to be really fiddly to do it like this for every form.

Maybe have a look at [overriding the built in widget templates](https://docs.djangoproject.com/en/5.2/ref/forms/renderers/) to call the macros